README.md

```diff
@@ -165,6 +165,20 @@ xe host-call-plugin host-uuid=<uuid>  plugin=smartctl.py fn=health
 {"/dev/sdf": "PASSED", "/dev/sdg": "PASSED", "/dev/sdd": "PASSED", "/dev/sde": "PASSED", "/dev/sdb": "PASSED", "/dev/sdc": "PASSED", "/dev/sda": "PASSED"}
 ```
 
+## IPMI-sensors parser
+A xapi plugin to get information about the host via the BMC
```

Comment:
Maybe add in the README that it requires `freeipmi` and `ipmitool`. I guess that it is already reported as an error if the plugin can not find them. What is the error if they are not installed? 
My point is that it can be cool, if it is not already the case, that when we run `xe host-call-plugin host-uuid=<uuid> plugin=2crsi-sensors.py fn="get_info"` the plugins return an clear message saying "ipmi-sensors not found". So just ensure that in case of error the user will understand that binary is not found.

Reply:
I can't remember, don't we have it already installed by default in XCP-ng?

Reply:
`ipmitool` is installed. Not `freeipmi`.